CHANGELOG.adoc

```diff
@@ -1,3 +1,28 @@
+=== Release 5.0.0 - Critical Design Review (CDR)
+
+* add battery temperature data to health packet 
+* add user story for requesting mission data
+* add user story 4.2 in derivedFrom in MissionData
+* replace user story 4.2 w/ request for mission data
+* add data structure for mission data
+* update verbiage in user story 4.1 example
+* add user story for requesting downlink via s-band
+* shorten example for user story 4.1
+* append detail to TLE data structure purpose
+* replace AODS IC with AODS GPS Data
+* fix tle_line_1 & tle_line_2 type in TLE
+* fix tle_data type in SatelliteHealth datastructure
+* update derivedFrom in ViewBeaconData user story
+* append detail to behavior in user story 2
+* remove user story 2.1
+* update derivedFrom in health data structure
+* update beacon to refer to health or mission data
+* replace payload user stories for scheduling with updating mission mode duration
+* update user story 3 to include any interrupt timer
+* bump major version no. in package.json
+* add user story for requesting to set MM duration
+* remove user stories 6-6 for payload scheduler
```

Comment:
Didn't you remove all the user stories starting with 5?

Reply:
Correct, which was then replaced w/ 5-SetMissionModeDuration.yaml